packages/adapter-node/src/handler.js

```diff
@@ -103,6 +104,56 @@ function serve_prerendered() {
 	};
 }
 
+/**
+ * @param {import('node:http').IncomingMessage} req
+ */
+function get_options(req) {
+	return {
+		platform: { req },
+		getClientAddress: () => {
```

Comment:
can we reduce the duplication here? there's another `getClientAddress` in this file, which looks quite similar